etc/set_plt.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e
+
+if [ `uname -m` = "armv6l" ]
```

Comment:
Don't hit JUST armv6l with this. Wildcard on armv* probably. Or arm* but not arm64